logstash-core/spec/api/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 # encoding: utf-8
 API_ROOT = File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "lib", "logstash", "api"))
 
+require "stud/task"
```

Comment:
code using is https://github.com/elastic/logstash/blob/master/logstash-core/spec/api/spec_helper.rb#L59, i only added this because I run the test with `bin/rspec` only for the api, not for everything. if I don't add the requirement here it does not work.